source/assets/javascripts/components/Leaderboards/ElevationLeaderboard/index.js

```diff
@@ -39,23 +42,23 @@ export default React.createClass({
     let data = response.results.map((result, index) => {
       let team = merge({}, result, result.team)
       team.id = team.team_page_id
-      team.distance_in_kms = team.distance_in_meters / 100
+      team.distance_in_kms = team.distance_in_meters / 1000
```

Comment:
I don't think this is used at all in Elevation? Might have been a copy paste error on my part?